lxd/containers_post.go

```diff
@@ -43,6 +43,8 @@ type containerImageSource struct {
 
 	/* for "copy" type */
 	Source string `json:"source"`
+	/* for "migration" type. Whether the migration is live. */
+	Live bool `json:"live"`
```

Comment:
Can't we make this a function with the implementation of `"criu" in self.Websockets`?